### ISSUE TITLE
Lock Down Date??

### ISSUE THREAD

**Issue by mcangeli** [RaspberryPints/RaspberryPints#139]
I saw it posted somewhere, but is there a formal post with the "Lock Down" date? 

I plan on grabbing a fresh copy and starting over and then testing the hell out of everything to help you all out and make sure that things are good.

**Comment by thadius856** [comment]
Thursday
13 Feb 2014
11:59 pm PST

**Comment by mcangeli** [comment]
Thank you.

Also, has a "dev forum" been setup where we can discuss things like wiring the flowmeters/extras, improvements, enhancements and the weather?

**Comment by thadius856** [comment]
Nope. Just here, and IRC so far.

**Comment by mtnbound** [comment]
The one feature I think we should add is the keg status tracking "on/off" config. I think a few people will just want to "add their beers and go".

It seems like a pretty simple change to add a config value, and then just remove the "where clause" on the tap drop down.

I'll work on it today. I saw a lot of discussion about which branch to use, but I'm not sure I got it all. Which one should I use now?
